### PR TITLE
Created more tests

### PR DESCRIPTION
**Pull request by mihikabairathi**
Added more tests from fluentd-catch-all-config and added a test for `@include` statements.

### CODE REVIEW

File: test/data/in_tail_chef.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
+log_file_path: /var/log/ops_agent/ops_agent.log
+logging_level: info
+logs_module:
+  sources:
+  - file_source_config:
+      checkpoint_file: /var/lib/google-fluentd/pos/chef-bookshelf.pos
+      parser:
```
Comment (StevenYCChou) [inline test/data/in_tail_chef.yaml:7]: If fluentd's format type is 'none', I think the tool doesn't add parser at all. Same for the rest of the file.

File: test/data/in_tail_include.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          expression: /^Started/
+        type: regex
+      path: /var/log/fluentd_test.log
+      refresh_interval: 60
+      rotate_wait: 5
+    name: test
+    type: file
+  - file_source_config:
+      parser:
+        type: none
```
Comment (StevenYCChou) [inline test/data/in_tail_include.yaml:30]: same here.

File: test/data/in_tail_rabbitmq.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+log_file_path: /var/log/ops_agent/ops_agent.log
+logging_level: info
+logs_module:
+  sources:
+  - file_source_config:
+      checkpoint_file: /var/lib/google-fluentd/pos/rabbitmq-startup.pos
+      parser:
+        type: none
```
Comment (StevenYCChou) [inline test/data/in_tail_rabbitmq.yaml:8]: same here.

File: test/data/in_tail_chef.yaml (hunk 1 above)
Comment (mihikabairathi) [reply test/data/in_tail_chef.yaml]: Oh I didn't know that - will change the logic.

File: test/test_config_converter.py [hunk 4]
```diff
@@ -27,6 +27,14 @@ def check_equality(config_name):
     assert expected == observed
 
 
+def test_types():
+    for prg in {
+            'config_script.py',
+            'config_converter/config_mapper/config_mapper.py'
+    }:
+        subprocess.run(['python3', '-B', '-m', 'pytype', prg], check=True)
```
Comment (StevenYCChou) [inline test/test_config_converter.py:35]: PyType tests! This is great that we can enforce the types for the functions with this test!

**Pull request merged**